### ISSUE TITLE
401: Unauthorized for all requests.

### ISSUE THREAD

**Issue by applabshp** [jarrodek/ChromeRestClient#650]
Hello,

I had multiple projects saved in old REST Client, when I upgraded to new one and use the requests from my old project, I get '401: Unauthorized' error. I use the same endpoint/headers/requests using SOAP UI and it works fine. Am I missing something here? Please let me know, appreciate it.

Thanks!

**Comment by jarrodek** [comment]
Hi,

It depends on which authentication method have you used in this requests.
Basically you missing Authentication header with auth data in it.
Since apps do not share the same network settings with Chrome I can't use Chrome's session management. It wasn't clear in Chrome apps docs when I updareded the apps. Now I'm implementing different auth methods on my own. It looks like almost nobody did it before for JavaScript. 
Basic authentication support is in the app. NTLM is under development, digest is planned for future releases. Which of this methods have you used before?

**Comment by sangohan** [comment]
the issue is that there are lots of webapps whose rest api uses cookie based auth. If the rest client doesn't use the same network session as the browser, it means it's impossible to use the rest client (unless i go copy all of the cookies into the rest client, which is a very annoying method to have to perform). 
The rest client prior to the new update works great for this particular scenario, and now it's broken. Chrome also forced my update, so i couldn't opt out until this issue is fixed. Rock and a hard place =(

**Comment by elipoultorak** [comment]
At least we should have a way to add cookies manually.

**Comment by jarrodek** [comment]
@elipoultorak I agree. There is a datastore for that, there is a cookie popup that could also have a "Save for future use"  checkbox when adding a cookie in headers editor. However, I'm working on different problems right now in the app. So if anyone could do it and send a PR it would be great. 
Also a possible option is to create an extension which will be used as a proxy between Chrome and the app so the app asks the extension for cookies for particular cookies and and the extension is providing this information back to the app. But it is too much of work for me right now and I don't have that much time to code it.

**Comment by jaeseok-park** [comment]
Hi, jarrodek,

I also met this issue too, and I've install ARC cookie exchanger.
However, I don't know how to use it.

I logged in my service, and posted an API on ARC app, but still 401 error.

Could you guide me how to use ARC cookie exchanger?

**Comment by jarrodek** [comment]
@jaeseok-park it's still in the develop branch. At this point it is released in [beta channel](https://groups.google.com/forum/#!forum/advanced-rest-client-beta-channel). When I have chance to test it properly (I'm out for Google I/O now) then I'll update stable version. I just don't want to release untested feature to public.

**Comment by jaeseok-park** [comment]
@jarrodek 
I understand you. I will wait for updating ARC.
Thank you.
